temporalcli/commandsgen/commands.yml

```diff
@@ -195,6 +195,10 @@ commands:
       - name: no-json-shorthand-payloads
         type: bool
         description: Raw payload output, even if they are JSON.
+      - name: rpc-timeout
+        type: duration
+        description: Timeout for all underlying RPC calls for the CLI.
+        default: 10s
```

Comment:
This should not have a default IMO. This timeout is for the entire command, not just one RPC call. A command may span many RPC calls.

Reply:
I also like "command timeout" or just "timeout" better than "rpc-timeout". If I am reading scripts, docs, etc. and I see `--rpc-timeout` with no other context, I will assume it's for a single RPC when it's for the entire command.